src/components/AddComment.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from "react";
+import { postComment } from "../utils/api";
+import { useParams } from "react-router-dom";
+import CommentList from "./CommentList";
+
+const AddComment = ({ setComments }) => {
+  const [author, setAuthor] = useState("");
+  const [body, setBody] = useState("");
+  const [check, setCheck] = useState(false);
+  const { article_id } = useParams();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if(body!=="" && author!=="") {
```

Comment:
Great misuse handling, the first thing I did was try leave a blank comment which you've handled well 😁